include/bitcoin/bitcoin/settings.hpp

```diff
@@ -31,6 +34,9 @@ class BC_API settings
 {
 public:
     settings();
+    settings(config::settings context);
+
+    void genesis(const std::string& encoded_genesis_block);
```

Comment:
Should be private.

Reply:
This will go away as you suggested below.